Annex60/Fluid/Movers/SpeedControlled_Nrpm.mo

```diff
@@ -16,7 +16,7 @@ model SpeedControlled_Nrpm
   parameter Real speed(final unit="1/min") = 0
     "Speed set point when using constant set point"
     annotation(Dialog(enable=inputType == Annex60.Fluid.Types.InputType.Constant));
-  parameter Real[:] speeds(each final unit="1/min") = {0}
+  parameter Real[:] speeds(each final unit="1/min") = per.speeds
```

Comment:
This should be `N`, or maybe better `Nrpm` to identify that this is not in a SI unit.